### ISSUE TITLE
Nested list (<ul>, <ol>) should be child of <li>, not sibling

### ISSUE THREAD

**Issue by karloku** [pavittarx/editorjs-html#49]
#### Expected behaviour

Given nested list as below

```markdown
- Green
  1. Lime
  2. Mint
- Red
  - Garnet
  - Crimson
- Blue
``` 

Should be parsed into

```html
<ul>
  <li>
    Green
    <ol>
      <li>Lime</li>
      <li>Mint</li>
    </ol>
  </li>
  <li>
    Red
    <ul>
      <li>Garnet</li>
      <li>Crimson</li>
    </ul>
  </li>
  <li>Blue</li>
</ul>
```

`<ul>` and `<ol>` should be child of `<li>`

#### Actual behaviour

`<ul>` and `<ol>` were parsed as siblings of `<li>`

```html
<ul>
  <li>Green</li>
  <ol>
    <li>Lime</li>
    <li>Mint</li>
  </ol>
  <li>Red</li>
  <ul>
    <li>Garnet</li>
    <li>Crimson</li>
  </ul>
  <li>Blue</li>
</ul>
```

This could cause some problems:

1. In W3C standards [`<ul>`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element) and [`<ol>`](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element) cound only have zero or more `<li>` as their content. Nesting `<ul>` and `<ol>` directly is not correct (would fail validation by [The Nu Html Checker](https://validator.w3.org/nu/#textarea)).
2. The official @editorjs/nested-list handles html ([`NestedList.pasteHandler`](https://github.com/editor-js/nested-list/blob/main/src/index.js#L208)) in the W3C way, making current result not handleable by the official NestedList component.

**Comment by Luzefiru** [comment]
This issue was resolved in #50.

**Comment by pavittarx** [comment]
Resolved in #49
